docs/team/zacwong2151.md

```diff
@@ -3,44 +3,49 @@ layout: default.md
+        * **Alternative 1 (current choice):** Saves both the volunteers and events state.
+        * **Alternative 2:** Selectively save either the volunteer or event storage.
+        * **Alternative 3:** Individual command knows how to undo/redo by itself.
+    * Credits: I used the high-level architecture of the undo/redo commands provided by the [SE-education AB4](https://se-education.org/addressbook-level4/DeveloperGuide.html#undo-redo-feature) as a reference to build the commands. 
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
 
-* **Code contributed**: [RepoSense link]()
 
-* **Project management**:
-    * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=zacwong&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code&since=2023-09-22&tabOpen=true&tabType=authorship&tabAuthor=zacwong2151&tabRepo=AY2324S1-CS2103T-T14-4%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
 
-* **Enhancements to existing features**:
-    * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-    * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+* **Project management**:
+    * Help set up the issue tracker and milestone management on GitHub
 
 * **Documentation**:
     * User Guide:
-        * Added documentation for the features `delete` and `find` [\#72]()
-        * Did cosmetic tweaks to existing documentation of features `clear`, `exit`: [\#74]()
+        * Added documentation for the features `efind`: [\#207](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/207) and `vfind`: [\#103](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/103)
+        * Added documentation for the features `undo` and `redo`: [\#206](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/206)
     * Developer Guide:
-        * Added implementation details of the `delete` feature.
+        * Added use cases for event delete feature: [\#67](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/67)
+        * Added implementation details for undo and redo features, inclusive of UML class diagrams: [#216](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/216)
+        * Added a Planned Enhancements section to propose fixes to add in the future: [\#215](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/215)
+        * Added a Instructions for Manual Testing section to give testers guidance on using our app: [\#218](https://github.com/AY2324S1-CS2103T-T14-4/tp/pull/218)
```

Comment:
Minor mistake: a Instructions

Reply:
my bro agrees with me that it shd be `a` not `an` which means `a` is prolly fine 😸

Reply:
becos its `a section` not `an section`

Reply:
okeh